packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx

```diff
@@ -0,0 +1,74 @@
+import { Component, h, JSX, Prop, Element, Fragment } from '@stencil/core';
+import { trackComponent } from '@utils/tracking/usage';
+import { logWarn } from '@utils/error/log-error';
+import { GuxAvatarSize } from './gux-avatar.types';
+
+/**
+ * @slot image - Headshot photo.
+ */
+
+@Component({
+  styleUrl: 'gux-avatar.scss',
+  tag: 'gux-avatar-beta',
+  shadow: true
+})
+export class GuxAvatar {
+  @Element()
+  root: HTMLElement;
+
+  /**
+   * Avatar size: small, medium or large
+   */
+  @Prop()
+  size: GuxAvatarSize = 'large';
+
+  /**
+   * Name which is shown as initials
+   */
+  @Prop()
+  name!: string;
+
+  private generateInitials(): string {
```

Comment:
Does this meet the requirements for other languages, such as Chinese or Japanese that don't use initials like we do in English?

Reply:
Conor confirmed with the localization team that this is will work fine once name is in correct format.